LEAF_Nexus/sources/Group.php

```diff
@@ -1082,6 +1082,25 @@ public function removePermission($groupID, $categoryID, $UID, $permissionType)
         return 0;
     }
 
+    /**
+     *
+     * @param string $tag_name
+     * @param int $groupID
+     *
+     * @return void
+     *
+     * Created at: 11/1/2022, 11:29:52 AM (America/New_York)
+     */
+    public function addGroupTag(string $tag_name, int $groupID): void
```

Comment:
Probably want to make sure neither of these are empty.

Reply:
The explicit calling of string and int will not allow it to come in empty. What I did do was made the check where the method is called, in System.php